HIP/hip-573.md

```diff
@@ -65,6 +65,11 @@ The network needs to include the new `all_collectors_are_exempt` field in state
 specification; and when deciding whether to assess a custom fee with `all_collectors_are_exempt = true`, needs to 
 check if the potential payer is a collector for the token type of that custom fee.
 
+### Mirror Node
+
+The mirror node is updated to add the new `all_collectors_are_exempt` field to the CustomFee in V1 and V2.
+This information will then be made available via new and existing REST APIs.
```

Comment:
This is too generic. We need to explicitly list which endpoints are being modified and what an example of the new JSON looks like.